src/Backups/BackupEntriesCollector.h

```diff
@@ -97,8 +98,24 @@ class BackupEntriesCollector : private boost::noncopyable
     std::shared_ptr<IBackupCoordination> backup_coordination;
     const ReadSettings read_settings;
     ContextPtr context;
-    std::chrono::milliseconds on_cluster_first_sync_timeout;
-    std::chrono::milliseconds consistent_metadata_snapshot_timeout;
+
+    const std::chrono::milliseconds on_cluster_first_sync_timeout;
```

Comment:
For my understanding, why do we need different timeout for first cluster sync ?

Reply:
The first cluster sync is a bit different. It means the time required to send the query to the cluster's nodes plus the time for the nodes to start working on the backup. By default `on_cluster_first_sync_timeout` is 180 milliseconds which corresponds to [distributed_ddl_task_timeout](https://github.com/ClickHouse/ClickHouse/blob/af8b1e4978519b23a8c550bd3df25a354ed799ca/src/Core/Settings.h#L305).

Reply:
I've added this comment to the code.